### ISSUE TITLE
Font name incorrect

### ISSUE THREAD

**Issue by casey-ac** [levidsmith/PythonProjects#1]
Attempting to play this game on a Debian based Linux distribution running Python 3.8.2 and Pygame 1.9.6 results in the following error because the font is called incorrectly. Changing the font name to end with ".ttf" instead of ".TTF" results in the game running correctly. 

pygame 1.9.6
Hello from the pygame community. https://www.pygame.org/contribute.html
Card Class
Traceback (most recent call last):
  File "/home/fienix/Downloads/PythonProjects-master/koikoi/koikoi.py", line 109, in <module>
    application = Application()
  File "/home/fienix/Downloads/PythonProjects-master/koikoi/koikoi.py", line 23, in __init__
    self.main()
  File "/home/fienix/Downloads/PythonProjects-master/koikoi/koikoi.py", line 39, in main
    font['small'] = pygame.font.Font("fonts/Seven Swordsmen BB.ttf", 20)
FileNotFoundError: [Errno 2] No such file or directory: 'fonts/Seven Swordsmen BB.ttf'

**Comment by levidsmith** [comment]
Good catch!  I hadn't tested under Linux yet.  Apparently in Windows case doesn't matter.

**Comment by levidsmith** [comment]
I forced the file extension to lowercase, which should solve that problem.

I tested it on Ubuntu 18.04 LTS, and also noticed an error when it plays sound effects.  I'm thinking that it just doesn't like WAV files in Linux, so I added an OS check and disabled sound effects if the operating system is not Windows.  I tried all of the various mixer.pre_init parameters.  If I get the time, I'll try changing them to OGGs or something else.  For now, it should play in Ubuntu Linux (without sound effects).
